### PR TITLE
Update releasing process doc.

### PR DESCRIPTION
**Pull request by randomvariable**
Update releasing docs for newcomers.
Bump cluster-api version for hack/tools

### CODE REVIEW

File: docs/releasing.md [hunk 1]
```diff
@@ -3,25 +3,22 @@
...
-### Release Notes
-
-Running `make release-notes` will generate an output that can be copied to the drafted release.
-Pay close attention to the `## :question: Sort these by hand` section, as it contains items that need to be manually sorted.
-
-
+9.  Perform the [image promotion process](https://github.com/kubernetes/k8s.io/tree/master/k8s.gcr.io#image-promoter).
+    The staging repository is at https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api-aws/GLOBAL. Be
+    sure to choose the top level `cluster-api-aws-controller`, which will provide the multi-arch manifest, rather than one for a specific architecture.
+1.  Write the release notes (see note below on release notes)
```
Comment (detiber) [inline docs/releasing.md]:
```suggestion
1.  Write the release notes
```

**Comment by randomvariable** [comment]
/hold
Need to add SELinux support (for building the binaries)

**Comment by randomvariable** [comment]
/unhold

we now support SELinux (for building binaries)

**Pull request merged**